lib/timeline/Core.js

```diff
@@ -183,22 +204,31 @@ Core.prototype._create = function (container) {
     // (else the page and timeline both scroll)
     event.preventDefault();
 
-    var delta = 0;
-    if (event.wheelDelta) { /* IE/Opera. */
-      delta = event.wheelDelta / 120;
-    } else if (event.detail) { /* Mozilla case. */
-      // In Mozilla, sign of delta is different than in IE.
-      // Also, delta is multiple of 3.
-      delta = -event.detail / 3;
-    }
-
-    var current = this.props.scrollTop;
-    var adjusted = current + delta * 120;
+    if (this.options.verticalScroll && Math.abs(deltaY) >= Math.abs(deltaX)) {
+      var current = this.props.scrollTop;
+      var adjusted = current + deltaY;
 
-    if (this.isActive()) {
-      this._setScrollTop(adjusted);
-      this._redraw();
-      this.emit('scroll', event);
+      if (this.isActive()) {
+        this._setScrollTop(adjusted);
+        this._redraw();
+        this.emit('scroll', event);
+      }
+    } else if (this.options.horizontalScroll) {
+      if (Math.abs(deltaX) >= Math.abs(deltaY)) var delta = deltaX;
```

Comment:
I would expect to have 
``` javascript
var delta;
if (Math.abs(deltaX) >= Math.abs(deltaY)) delta = deltaX;
else delta = deltaY;
```

It is working when the delta `var` is only defined in the `if` scopes and then used in a different scope, because this is one of those "bad parts of JS". ES6 won't let this work... I'm afraid this is bad practice in general and will be a pitfall when we upgrade to es6.